src/index.html

```diff
@@ -11,7 +11,175 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="box-container">
+        <legend>Personal information</legend>
+        <label 
+        class="form-label" 
+        for="surname">
+          Surname:
+          <input 
+            type="text" 
+            autocomplete="off" 
+            name="surname" 
```

Comment:
again, you seem to be trying to link label to an input via an `<input ... name` attribute - it doesn't work this way.
`<label ... for` is linked to `<input ... id` attribute:
```suggestion
        for="surname">
          Surname:
          <input 
            type="text" 
            autocomplete="off" 
            id="surname" 
            name="surname" 
```